encoding/typescript/gen.go

```diff
@@ -64,41 +64,38 @@ func GenerateTypes(sch thema.Schema, cfg *TypeConfig) (*ast.File, error) {
 	}
 	if cfg.CuetsyConfig == nil {
 		cfg.CuetsyConfig = &cuetsy.Config{
+			ImportMapper: func(_ string) (string, error) {
+				return "", nil
+			},
```

Comment:
```suggestion
			ImportMapper: cuetsy.IgnoreImportMapper,
```

Reply:
I think this should work, it's the same, but [defined in Cuetsy](https://github.com/grafana/cuetsy/blob/main/import_mapper.go#L27C6-L27C24).

Reply:
I didn't recall that we had that!